.github/workflows/ci-cd.yml

```diff
@@ -23,6 +26,9 @@ jobs:
       - name: Install dependencies
         run: cd astro_site && npm ci
 
+      - name: Run microformats tests
+        run: cd astro_site && npm run test:microformats
+
       - name: Install ImageMagick for HEIC conversion
         run: sudo apt-get update && sudo apt-get install -y imagemagick
```

Comment:
`Run microformats tests` needs to run after `Install ImageMagick for HEIC conversion`